tests/queries/0_stateless/02415_all_new_functions_must_be_documented.reference

```diff
@@ -56,6 +56,10 @@ URLHierarchy
 URLPathHierarchy
 UUIDNumToString
 UUIDStringToNum
+YYYYMMDDToDate
+YYYYMMDDToDate32
+YYYYMMDDhhmmssToDateTime
+YYYYMMDDhhmmssToDateTime64
```

Comment:
I don't think we should edit this file, as the name says, we should add documentation for all new functions in the code using `FunctionDocumentation argument in factory.registerFunction` . Let's do it for these new functions